src/Pages/gunwookim/Login/Login.js

```diff
@@ -1,8 +1,66 @@
 import React from 'react';
 
+import { Link } from 'react-router-dom';
+
+import './Login.scss';
+
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      isVerify: false,
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value }, this.loginValidation);
+  };
+
+  loginValidation = () => {
+    const { email, password } = this.state;
+    const emailVerify = email.includes('@');
+    const passwordVerify = password.length >= 5 ? true : false;
+
+    this.setState({
+      isVerify: emailVerify && passwordVerify,
```

Comment:
```suggestion
      isVerified: isPasswordVerified,
```
- 위에서 boolean 타입임을 확인했고, isVerified는 초기값으로 boolean을 선언해줬기 때문에, 이후에 들어올 값도 같은 타입이면 좋겠습니다. `&&` 연산자의 경우 false로 평가되는 경우 null을 리턴하기 때문에 굳이 이 방법 대신 `isPasswordVerified`를 써주는 건 어떨까요?
- 그리고 애초에 isPasswordVerified이름을 isVerified로 써주면 객체 단축 속성명에 의해 다음과 같이 사용할 수 있습니다.
```js
this.setState({ isVerified })
```